Source/engine.cpp

```diff
@@ -42,7 +42,7 @@ void CelDrawTo(CelOutputBuffer out, int sx, int sy, BYTE *pCelBuff, int nCel, in
 	CelBlitSafeTo(out, sx, sy, pRLEBytes, nDataSize, nWidth);
 }
 
-void CelClippedDrawTo(CelOutputBuffer out, int sx, int sy, BYTE *pCelBuff, int nCel, int nWidth)
+void CelClippedDrawTo(CelOutputBuffer out, int sx, int sy, BYTE* pCelBuff, int nCel, int nWidth)

```

Comment:
Can we undo this to reduce the diff? Unrelated change.

Reply:
Done. I don't remember doing this change. It must have been done automatically by VS2019.

Reply:
see if yo can get it to follow thw .clang-format configuration in the project